src/CommonSpaces/CommonSpaces.jl

```diff
@@ -0,0 +1,199 @@
+"""
+    CommonSpaces
+
+CommonSpaces contains convenience constructors for common
+spaces, which builds off of [`CommonGrids`](@ref) and
+(when appropriate) requires an additional argument,
+`staggering::Staggering` to construct the desired
+space.
+"""
+module CommonSpaces
+
+export ExtrudedCubedSphereSpace,
+    CubedSphereSpace, ColumnSpace, Box3DSpace, SliceXZSpace, RectangleXYSpace
+
+export Grids
+import ..Grids
+import ..Grids: Staggering
+import ..Spaces
+import ..CommonGrids
+import ..CommonGrids:
+    ExtrudedCubedSphereGrid,
+    CubedSphereGrid,
+    ColumnGrid,
+    Box3DGrid,
+    SliceXZGrid,
+    RectangleXYGrid
+
+"""
+	ExtrudedCubedSphereSpace(args...; staggering::Staggering, kwargs...)
+
+Calls [`CommonGrids.ExtrudedCubedSphereGrid`](@ref),
+with the args, kwargs, plus an additional `staggering::Staggering`
+keyword, to construct a cell center or cell face space.
```

Comment:
I suggest we rephrase this along the lines of:

```
Construct an [`ExtrudedFiniteDifferenceSpace`](@ref) for a cubed sphere configuration.

To make the Space face-centered, pass `staggering = Grids.FaceCenter()`.

You can customize various properties of the space, including resolution, number of elements, radius, et cetera, by passing the relative argument. Refer to [`CommonGrids.ExtrudedCubedSphereGrid`](@ref) for the full list of arguments
```

The key point of this function is not that it calls `CommonGrids.ExtrudedCubedSphereGrid`, but that it constructs a `Space`.